src/config/config.rs

```diff
@@ -108,6 +114,21 @@ impl DatabaseSetting {
   }
 }
 
+impl ToString for DatabaseSetting {
+  /// Converts [DatabaseSetting] into its postgres connection URL.
+  fn to_string(&self) -> String {
```

Comment:
Using `to_string` for converting a database connection structure into a PostgreSQL URL might indeed be too vague and not self-explanatory. Creating a dedicated function, such as `to_pg_url`, would be a clearer and more descriptive approach.